deploy/examples/common.yaml

```diff
@@ -160,18 +160,6 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get"]
-  - apiGroups: ["replication.storage.openshift.io"]
```

Comment:
RBACs are still required isn't it?

Reply:
yes, we need it. Forgot to revert the change. Will update it.

Reply:
Looks like RBAC is required only for the controller, and controller RBACs are installed separately we can remove this one but need testing and confirmation